src/userscores/qml/MuseScore/UserScores/internal/ExportOptionsView.qml

```diff
@@ -0,0 +1,251 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Dialogs 1.2
+
+import MuseScore.UserScores 1.0
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+
+ColumnLayout {
+    required property ExportScoreModel scoresModel
+    required property ExportScoreSuffixModel suffixModel
+    required property ExportScoreSettingsModel settingsModel
+
+    spacing: 12
+
+    Column {
+        spacing: 6
+        Layout.fillWidth: true
+
+        StyledTextLabel {
+            Layout.fillWidth: true
+            text: qsTrc("userscores", "Export To:")
+            font.capitalization: Font.AllUppercase
+        }
+
+        Row {
```

Comment:
Recently I have implemented a new component for this: FilePicker (it is already in the master). You should use it here

Reply:
The design will be a bit different (the location will be asked when the user clicks the "Export" button). 
(Otherwise FilePicker would be great here.)